### PR TITLE
Added testing label

### PR DESCRIPTION
**Pull request by CunningDJ**
Added much needed labeling to tests.  Please merge!

### CODE REVIEW

File: area.py [hunk 1]
```diff
@@ -9,6 +9,7 @@ def area_of_rect(x, y):
 def area_of_right_triangle(side1, side2):
     return (side1*side2) / 2
 
+print('TESTING STUFF:')
```
Comment (jarrodparkes) [inline area.py]: sweet!

Comment (CunningDJ) [inline area.py]: Awesome. (me)

**Comment by jarrodparkes** [comment]
@CunningDJ can you add labels for each test?

**Comment by jarrodparkes** [comment]
looks good! merging!

**Pull request merged**